### ISSUE TITLE
Show help message when invoke command without argument

### ISSUE THREAD

**Issue by conao3** [markuplint/markuplint#87]
In general, when the user directly invokes a command that requires an argument, it will often output a simple help along with an indication that an argument is required.

I discovered the --help option on a hunch, but it's not friendly.
Please show a help message with a warning that an argument is required when the user invokes `markuplint` it without any argument?

## npm without argument
```
dbbf47d7060b% npm

Usage: npm <command>

where <command> is one of:
    access, adduser, audit, bin, bugs, c, cache, ci, cit,
    clean-install, clean-install-test, completion, config,
    create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
    edit, explore, fund, get, help, help-search, hook, i, init,
    install, install-ci-test, install-test, it, link, list, ln,
    login, logout, ls, org, outdated, owner, pack, ping, prefix,
    profile, prune, publish, rb, rebuild, repo, restart, root,
    run, run-script, s, se, search, set, shrinkwrap, star,
    stars, start, stop, t, team, test, token, tst, un,
    uninstall, unpublish, unstar, up, update, v, version, view,
    whoami

npm <command> -h  quick help on <command>
npm -l            display full usage info
npm help <term>   search for help on <term>
npm help npm      involved overview

Specify configs in the ini-formatted file:
    /home/conao/.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@6.14.8 /home/conao/.anyenv/envs/nodenv/versions/14.13.0/lib/node_modules/npm
```
## markuplint without argument
(nothing output)
```
dbbf47d7060b% npx markuplint       
dbbf47d7060b% 
```

**Comment by YusukeHirao** [comment]
Thanks, @conao3. It's my fault, I overlooked.

**Comment by YusukeHirao** [comment]
Fixed on `v1.0.1`

Thanks to @conao3
